services-directions-models/src/main/java/com/mapbox/api/directions/v5/utils/FormatUtils.java

```diff
@@ -254,4 +262,23 @@ public static String formatPointsList(@Nullable List<Point> points) {
     }
     return join(";", coordinatesToJoin);
   }
+
+  private static DateTimeFormatter provideDateTimeFormat() {
+    return DateTimeFormatter.ofPattern(ISO_8601_PATTERN);
+  }
```

Comment:
do we need a separate method for a formatter?
There is only a single line of code, so I would prefer just to use
`DateTimeFormatter.ofPattern(ISO_8601_PATTERN)`inside `formatDateTimeToIso8601 ` method
or we might have an input param to save a method and make it more flexible

```
private static DateTimeFormatter provideDateTimeFormatter(String format) {
    return DateTimeFormatter.ofPattern(format);
  }
```

Reply:
Seems that it's redundant with implementation using `java.util.Date`. I'll move body to method